pkgs/top-level/all-packages.nix

```diff
@@ -20510,6 +20510,17 @@ in
 
   syncthing-tray = callPackage ../applications/misc/syncthing-tray { };
 
+  syncthingtray-minimal = libsForQt5.callPackage ../applications/misc/syncthingtray { };
```

Comment:
Do we actually need the minimal version? It will just lead to duplicate builds on hydra.

Also is this package actually different from `syncthing-tray`?

Reply:
> Also is this package actually different from `syncthing-tray`?

Oh yes it's different. I wonder how come all of these developers implemented these similar applications but I can justify `@Marcthus` since I think his application is the best.

> Do we actually need the minimal version? It will just lead to duplicate builds on hydra.

Personally, I don't have any use in the non minimal version as it includes all of the file browsers' integration and some users (like me) who don't use desktop environments with full fledged file browser don't need this integration compiled.

However, I think that users who do have such a file browser will expect these features to be compiled as well.

Reply:
Fair enough.

Reply:
> I wonder how come all of these developers implemented these similar applications

I thought it was easier to start from scratch. At least qsyncthingtray contained some bad code (in my opinion) and the other projects were not very sophisticated anyways or had another focus.

> I think his application is the best

Thanks :-)